src/plugins/shared_ux/public/components/redirect_app_links/redirect_app_links.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { FunctionComponent, useRef, useMemo } from 'react';
+import useObservable from 'react-use/lib/useObservable';
+import { ApplicationStart } from 'src/core/public';
+import { createNavigateToUrlClickHandler } from './click_handler';
+
+export interface RedirectAppLinksProps extends React.HTMLAttributes<HTMLDivElement> {
+  navigateToUrl: ApplicationStart['navigateToUrl'];
```

Comment:
I'm not going to block the PR, but we'll need to refactor this to not rely on other plugins or services, even `core`.  Feel free to add a TODO and assign it to me, as I'm working on this in the immediate term.

Reply:
I opened https://github.com/elastic/kibana/issues/127464 to track this thanks